Page/cart/cart.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/style.css">
+    <link rel="stylesheet" href="cart.css">
+    <script src="js/fontawesomepro.js"></script>
+    <link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.0.0/css/all.css">
+    <link rel="stylesheet" href="https://mango-ui.netlify.app/components/card/card.css" />
+    <link rel="stylesheet" href="https://mango-ui.netlify.app/components/button/button.css" />
+    <title>Dope Kicks</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav-container">
+            <div class="nav-logo"><a href="index.html">Dope Kicks <i class="fa-solid fa-boot"></i></a></div>
+            <input type="search" placeholder=" Search items here" class="nav-input">
+            <button class="nav-btn"><a href="#">Home</a></button>
+            <a class="cart-icon" href="/page/cart/cart.html"><i class="fa-solid fa-cart-shopping-fast"></i></a>
+            <a class="cart-icon" href="/page/logout/logout.html"><i class="fa-solid fa-right-from-bracket"></i></a>
+
+        </nav>
+
+        <main>
```

Comment:
`header` element should end here. 

Don't nest main inside header

Reply:
done